spec/features/admin/index_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
```

Comment:
We may want to name this something admin_merchants_index_spec.rb

Reply:
Yeah, I think we could easily float this over to the existing admin_merchants_spec.rb